### ISSUE TITLE
get_version_info should be updated to be aware of new voldrv naming (deduped/non-deduped)

### ISSUE THREAD

**Issue by pploegaert** [openvstorage/framework#778]
2016-08-05 11:42:02 49100 +0200 - ovs52n252 - 125179/140635645032256 - celery/celery.worker.job - 1294 - INFO - Task ovs.storagerouter.get_support_info[93c0ac06-d98d-4076-8c9f-89193fdd0828] succeeded in 1.245151471s: {'enabled': True, 'enablesupport': False, 'clusterid': u'uO8JiRQaLdOqcw9D', 'nodeid': 'Ut6KtRhuKTp4CjVd',...
dpkg-query: package 'volumedriver-server' is not installed and no information is available
Use dpkg --info (= dpkg-deb --info) to examine archive files,
and dpkg --contents (= dpkg-deb --contents) to list their contents.
dpkg-query: package 'volumedriver-base' is not installed and no information is available
Use dpkg --info (= dpkg-deb --info) to examine archive files,
and dpkg --contents (= dpkg-deb --contents) to list their contents.

**Comment by wimpers** [comment]
@khenderick isn't this a duplicate of https://github.com/openvstorage/framework/issues/649 which is set to closed.

**Comment by khenderick** [comment]
@wimpers, the previous ticket added support for the various flavors of volumedriver, but left the dirty looking messages in the logging. However, I would use this ticket to cleanup the messy log entries.

**Comment by khenderick** [comment]
Fixed by #1040, packaged in openvstorage-2.7.4-rev.4149.6a15e4f

**Comment by kinvaris** [comment]
PASSED:

```
# before the fix
In [6]: subprocess.check_output("dpkg -s openvstorage | grep Version | cut -d ' ' -f 2", shell=True)
Out[6]: '2.7.3-fargo.1-1\n'

In [7]: subprocess.check_output("dpkg -s openvstorage1 | grep Version | cut -d ' ' -f 2", shell=True)
dpkg-query: package 'openvstorage1' is not installed and no information is available
Use dpkg --info (= dpkg-deb --info) to examine archive files,
and dpkg --contents (= dpkg-deb --contents) to list their contents.
Out[7]: ''

# after the fix
In [8]: subprocess.check_output("dpkg -s openvstorage 2> /dev/null | grep Version | cut -d ' ' -f 2", shell=True)
Out[8]: '2.7.3-fargo.1-1\n'

In [9]: subprocess.check_output("dpkg -s openvstorage1 2> /dev/null | grep Version | cut -d ' ' -f 2", shell=True)
Out[9]: ''
```
